package-lock.json

```diff
@@ -13,21 +13,9 @@
       }
```

Comment:
We shouldn't have a `package-lock.json` file since we are using yarn as our package manager - this means that you should do `rm package-lock.json` commit the deletion of the file, and then run `yarn install` to ensure that `yarn.lock` is up to date!